addons/survey/views/survey_templates.xml

```diff
@@ -53,14 +62,15 @@
                 t-att-data-timer="answer.start_datetime.isoformat() if survey.is_time_limited
                     and answer.start_datetime and answer.state != 'done' else False"
                 t-att-data-time-limit-minutes="survey.time_limit"
-                t-att-data-is-start-screen="answer.state == 'new'">
+                t-att-data-is-start-screen="answer.state == 'new'"
+                t-att-data-questions-layout="survey.questions_layout">
             <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
             <input type="hidden" name="token" t-att-value="answer.access_token" />
             <div class="o_survey_error alert alert-danger d-none" role="alert">
                 <p>There was an error during the validation of the survey.</p>
             </div>
 
-            <div class="o_survey_form_content">
+            <div t-att-class="'o_survey_form_content' + (' mt128 ' if survey.questions_layout == 'page_per_question' else '')">
```

Comment:
Can you use %s notation instead of string concatenation?
Should be a python evaluated template right?